infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/ServerServiceBuilder.java

```diff
@@ -63,7 +63,7 @@ public Service build() {
         new io.fabric8.kubernetes.api.model.ServiceBuilder();
     return builder
         .withNewMetadata()
-        .withName(name.replace("/", "-"))
+        .withName(name.replace("/", "-").toLowerCase())
```

Comment:
what is the reason to do ```toLowerCase```?

Reply:
Plugin broker can (and actually does) submit value which contains upper case chars which are not supported in k8s